test/functional/api/v2/job_invocations_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_plugin_helper'
```

Comment:
we use rspec syntax in unit tests and this one in functional so far (job_templates_controller_test.rb), I think we should start using just one